<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registre</class>
 <widget class="QDialog" name="registre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="reg_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>register</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>+statut</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>301</width>
     <height>248</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit_3cne"/>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lineEdit_5filiere"/>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit_1nom"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Filiere</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLineEdit" name="lineEdit_7username"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_2prenom"/>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_8password">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>date</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>CNE</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEdit_4cin"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
